### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by mAAdhaTTah** [roots/trellis#1098]
* [X] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
* [X] This request isn't a duplicate of an existing issue
* [X] I've read the [docs](https://roots.io/documentation/) and followed them (if applicable)
* [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

## Description

Getting this deprecation warning when running provision:

> TASK [swapfile : install dependencies] ************************************************************************************************
> [DEPRECATION WARNING]: Invoking "apt" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply
 multiple items and specifying `name: "{{ item }}"`, please use `name: '{{ swapfile_dependencies }}'` and remove the loop. This
feature will be removed in version 2.11. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
ok: [159.203.148.200] => (item=['util-linux'])

## Steps to reproduce

1. Run `ansible-playbook provision`.

**Expected behavior:** [What you expect to happen]

No deprecation warnings.

**Actual behavior:** [What actually happens]

Shows a deprecation warning.

**Reproduces how often:** [What percentage of the time does it reproduce?]

Every time.

## Versions

Ansible 2.7.10 (just outside the supported range, but everything else still works)
